src/components/add-comment/add-comment.jsx

```diff
@@ -0,0 +1,58 @@
+import { useState } from "react";
+import { useParams } from "react-router-dom";
+import * as api from '../../utils/api';
+import './add-comment.css';
+ 
+const AddComment = () => {
+
+    const {id} = useParams()
+    
+    const [addCommentUsername, setAddCommentUsername] = useState("")
+    const [addCommentBody, setAddCommentBody] = useState("")
+    const [addComment, setAddComment] = useState("")
+ 
+    const handleSubmit = (event) => {
+        const commentToAdd = {
+            username: addCommentUsername,
+            body: addCommentBody
+        };

```

Comment:
We need some feedback that its worked. 
Either the comment appears at the top, or some kind of success message please. 
its hard to tell if its been posted at the moment, without scrolling and looking around